src/CrawlAbstractLvl/private/obstaclered.cpp

```diff
@@ -7,12 +7,18 @@
 
 #include "obstaclered.h"
 
-ObstacleRed::ObstacleRed() : IWorldItem("ObstacleRed") {
+namespace AbstractLvl {
+
+ObstacleRed::ObstacleRed() : IWorldItem(AUTO_CLASS_NAME) {
     setMash("qrc:/mesh/meshes/ObstacleRed.mesh");
-    setSize({1,1,1});
+    setSize({2,2,2});
```

Comment:
```suggestion
    setSize({1,1,1});
```
please use the the 1 as a scale factor. if you think that object is small then please create new task for the @Oleg-designer